features/diffing.feature

```diff
@@ -25,13 +25,60 @@ Feature: diffing
     Then the output should contain:
       """
              Diff:
-             @@ -1,4 +1,4 @@
+             @@ -1,5 +1,5 @@
               this is the
              -  expected
              +  actual
                   string
       """
 
+  Scenario: diff for a multiline string with trailing newline
+    Given a file named "example_spec.rb" with:
+      """ruby
+      RSpec.describe "a multiline string" do
+        it "is like another string" do
+          expected = "Chewy\nbagel."
+          actual = "Chewy\nbagel.\n"
+          expect(actual).to eq(expected)
+        end
+      end
+      """
+    When I run `rspec example_spec.rb`
+    Then the output should contain:
+      """
+             Diff:
+             @@ -1,3 +1,4 @@
+              Chewy
+              bagel.
+             +
```

Comment:
Hm, do we want a `+` or `-` in front of `\ No newline at end of input` depending on how it relates to the expected?

Also why I thought maybe just a `+` or `-` might be acceptable:
<img width="534" alt="screen shot 2016-07-06 at 9 16 30 pm" src="https://cloud.githubusercontent.com/assets/3820966/16639600/f35c0dfe-43be-11e6-8f63-649f403975ab.png">
<img width="537" alt="screen shot 2016-07-06 at 9 15 53 pm" src="https://cloud.githubusercontent.com/assets/3820966/16639603/f545c650-43be-11e6-9857-a540f5a6e6ac.png">